src/components/ShowAddCard/AddCardForm/AddCardForm.test.js

```diff
@@ -0,0 +1,46 @@
+import { shallow } from 'enzyme';
+import React from 'react';
+import { AddCardForm } from './AddCardForm';
+
+describe('<ShowAddCard />', () => {
+  let wrapper;
+  const props = {
+    isAddCardVisible: false,
+    handleAddCard: jest.fn(),
+  };
+  beforeEach(() => {
+    wrapper = shallow(<AddCardForm {...props} />);
+  });
+
+  it('should render without crash', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+
+  it('should state be empty strings', () => {
```

Comment:
As setting initial state is hardcoded and does not contain logic, we do not need to test it